packages/rolldown/src/cli/index.ts

```diff
@@ -1,55 +1,34 @@
 import process from 'node:process'
-import parseArgs from 'mri'
-import { defineCommand, runMain, showUsage } from 'citty'
+import { parseArgs } from 'node:util'
+import { mapValues, omit } from 'remeda'
 import { bundle } from './commands/bundle'
-import {
-  version,
-  description,
-} from '../../package.json' assert { type: 'json' }
+import { showHelp } from './commands/help'
 import { DEFAULT_CONFIG_FILENAME } from './constants'
+import { CLI_OPTIONS } from './options'
```

Comment:
The value should not be static when the automatically generated arguments are implemented, so I suggest using the `camelCase` method instead of the UPPER SNAKE_CASE method.

Reply:
In this PR, the value is still static; you can change it to camelCase in your subsequent PR.

Reply:
Oh, I see.